resume-input/README.md

```diff
@@ -1 +1,5 @@
-#The Idea
```

Comment:
Great job with the readme, so many people do not create a readme, or they leave the general READme that is created with the library that they are using